package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "app_scaffold",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Should probably be a major version bump, considering we're dropping support for older node/ruby versions?

Reply:
Sure, I can update it to be 2.0.0. :)

Reply:
Addressed :)